jsons/messages/it.json

```diff
@@ -0,0 +1,12 @@
+{
+  "signatory_not_found":"Firmatario non trovato",
+  "signatory_already_confirmed":", avete già firmato il Manifesto per l'Artigianato del Software.",
```

Comment:
You are using the second plural person, is there a reason? 
If no, I will just use the second singular person, so in this case "Hai già firmato" and below there are several:
- "Vorresti firmare il Manifesto..." 
- "Ora sei un firmatario..."
- "Hai rifiutato di firmare..."

Reply:
You're right, it was due to my last three years passed with french speaking people. That influenced a bit my culture.

They do use that person in a professional and formal context. Sure, in Italy we use more ofte the second singular person :)